x/ccv/provider/keeper/power_shaping.go

```diff
@@ -101,10 +102,6 @@ func (k Keeper) CapValidatorSet(
 
 	validatorSetCap := powerShapingParameters.ValidatorSetCap
 	if validatorSetCap != 0 && int(validatorSetCap) < len(validators) {
-		sort.Slice(validators, func(i, j int) bool {
```

Comment:
Explain in a comment why it's safe to remove the sort, i.e., the validators are already in the correct order.

Reply:
Sorry, I meant add the comment to the code :)

Reply:
I added a shorter one just in case, so it's in the code too.